openshift/dockerfiles/cache.Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG GO_BUILDER=brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.22
+ARG RUNTIME=registry.access.redhat.com/ubi9/ubi-minimal:latest@sha256:c0e70387664f30cd9cf2795b547e4a9a51002c44a4a86aa9335ab030134bf392
+
+FROM $GO_BUILDER AS builder
+
+WORKDIR /go/src/github.com/openshift-pipelines/tekton-caches
+COPY . .
+
+RUN go build -v -o /tmp/cache  ./cmd/cache
+
+FROM $RUNTIME
+ARG VERSION=tekton-caches-main
+
+COPY --from=builder /tmp/cache /ko-app/cache
```

Comment:
We should do the same we do for things like git-init or entrypoint (`shadow-utils`, …).
- https://github.com/openshift-pipelines/ecosystem-images/blob/main/git-init/Dockerfile#L32
- https://github.com/openshift-pipelines/tektoncd-pipeline/blob/main/openshift/dockerfiles/entrypoint.Dockerfile#L37

Reply:
cache images is executed by script via stepAction.
do we still need an entry point for this ?

Reply:
I don't mean the entrypoint, I mean installing `shadow-utils` and doing the addgroup, … that we do in other images.